go/tools/builders/checker_main.go

```diff
@@ -44,39 +44,93 @@ import (
 // do not return an error so as not to unnecessarily interrupt builds.
 func run(args []string) error {
 	archiveFiles := multiFlag{}
+	importMapIn := multiFlag{}
+	srcs := multiFlag{}
 	flags := flag.NewFlagSet("checker", flag.ContinueOnError)
 	flags.Var(&archiveFiles, "archivefile", "Archive file of a direct dependency")
+	flags.Var(&importMapIn, "importmap", "Import maps of a direct dependency")
```

Comment:
Continuing the thread about importcfg files, Bazel will set `GOROOT` to the *target* standard library root, which is what we need. That's what the code being checked will compile and link against.

I don't think the I/O overhead will be bad. The file will be freshly written by compile.go, so it will definitely be in the kernel cache. I'd be mildly surprised if that data ever actually hits the disk.

If you want to do this in another PR after `master` is merged to `feature/check`, that's fine, but please add a TODO here.

Reply:
Ack. I've added a TODO here which I will address after merging `master` into `feature/check`.